circle.yml

```diff
@@ -58,7 +58,7 @@ version: 2
 jobs:
   generic:
     docker:
-      - image: balenalib/amd64-alpine-node:10
+      - image: balenalib/amd64-alpine-node:12
```

Comment:
This is required for the tests to pass since the supervisor requires node 12 now - no idea how this didn't fail on previous PRs